src/ATen/native/xpu/mkl/SpectralOps.cpp

```diff
@@ -399,8 +369,8 @@ Tensor& _fft_c2c_mkl_out(
   auto result = _fft_c2c_mkl(
       self, dim, static_cast<int64_t>(fft_norm_mode::none), forward);
   at::native::resize_output(out, result.sizes());
-  return impl::_fft_apply_normalization_out(
-      out, result, normalization, result.sizes(), dim);
+  out.copy_(result);
+  return out;
```

Comment:
The semantics before and after modification are not equivalent. Please note that the `normalization` parameter above is `fft_norm_mode::none`. Seems you want to apply `auto result = _fft_c2c_mkl(self, dim, normalization, forward);`, right?